starknet_py/net/full_node_client.py

```diff
@@ -398,6 +398,14 @@ async def get_pending_transactions(self) -> List[Transaction]:
 
         return PendingTransactionsSchema().load(res, unknown=EXCLUDE)
 
+    async def get_nonce(
+        self,
+        contract_address: int,
+        block_hash: Optional[Union[Hash, Tag]] = None,
+        block_number: Optional[Union[int, Tag]] = None,
+    ) -> int:
+        raise NotImplementedError
```

Comment:
We can use it already https://github.com/starkware-libs/starknet-specs/blob/v0.1.0/api/starknet_api_openrpc.json#L629